spec/spec_helper.rb

```diff
@@ -6,13 +8,15 @@
   add_filter 'spec/'
 end
 
+Warning[:deprecated] = true if defined?(Warning.[]=)
```

Comment:
Nice PR @BrianHawley 
What is the use of this line about deprecated ?

Reply:
That makes Ruby deprecation warnings show up while running tests. I used those warnings to determine and fix any future compatibility issues, when I was running the tests locally against all supported versions. It's a good trick I use at work as well.